docs/source/a_quick_tour.mdx

```diff
@@ -239,7 +239,7 @@ evaluate.push_to_hub(
   dataset_type="wikitext",                # dataset name on the hub
   dataset_name="WikiText",                # pretty name
   dataset_split="test",                   # dataset split used
-  task_type="text-generation",            # task id, see https://github.com/huggingface/datasets/blob/master/src/datasets/utils/resources/tasks.json
+  task_type="text-generation",            # task id, see https://github.com/huggingface/evaluate/blob/7fa754cc38c523b69054c67b8c3b8e3f4962fbf6/src/evaluate/config.py#L154-L192
```

Comment:
The config doesn't change much so let's go for main :) Otherwise looks good!

```suggestion
  task_type="text-generation",            # task id, see https://github.com/huggingface/evaluate/blob/main/src/evaluate/config.py#L154-L192
```

Reply:
ok! I've committed your suggestion, thanks 😄